lib/tools/createToken.ts

```diff
@@ -20,43 +19,46 @@ const createToken = (question: string) => tool({
     mimeType: z.string().optional().describe("The type of media."),
   }),
   execute: async ({ image, title, collectionAddress, mimeType }) => {
-    if (!image || !mimeType) {
+    const validationErrors = [
+      { condition: !auth.authenticated, status: CreateTokenResponse.MISSING_AUTH, message: "Please connect your wallet first." },
+      { condition: !auth.user?.wallet?.address, status: CreateTokenResponse.MISSING_WALLET, message: "No wallet address found." },
```

Comment:
Question: ❓ 
Can we pass only address from privy to check MISSING_AUTH & MISSING_WALLET?
Tool call param: image, title, mimeType, collectionAddress, walletAddress
Status: MISSING_CONNECTED_WALLET

  `const { login, user } = usePrivy();
  const address = user?.wallet?.address as Address;`